composer.json

```diff
@@ -47,6 +47,7 @@
         "symfony/config": "^7.0",
         "symfony/console": "^7.0",
         "symfony/dom-crawler": "^7.0",
+        "symfony/event-dispatcher": "^7.0",
```

Comment:
Is this package directly used in framework? If not, we should use `conflicts` against `>=8.0.0` instead.

Reply:
In fact it's probably safest to do that anyway - just in case someone is being weird and using `6.x` in their own project/module explicitly - we don't want to break their dependencies when we don't need to.

Reply:
I would have still preferred `conflicts` but this is better than just leaving it broken so will merge.